tests/analyses/reaching_definitions/test_subject.py

```diff
@@ -54,23 +57,30 @@ def test_when_instanciated_with_a_function_need_other_attributes(_, __):
 
     subject = Subject(function, func_graph, cc)
 
-    nose.tools.assert_equals(subject.func_graph, func_graph)
-    nose.tools.assert_equals(subject.cc, cc)
+    assert (subject.func_graph == func_graph)
+    assert (subject.cc == cc)
 
 
 def test_cc_attribute_should_raise_error_when_subject_is_a_block():
     arch = ArchX86()
     block = Block(0x42, byte_string=b'', arch=arch)
     subject = Subject(block)
-
-    with nose.tools.assert_raises(TypeError):
+    flag = False
+    try:
         _ = subject.cc
+    except TypeError:
+        flag = True
+    assert flag
 
 
 def test_func_graph_attribute_should_raise_error_when_subject_is_a_block():
     arch = ArchX86()
     block = Block(0x42, byte_string=b'', arch=arch)
     subject = Subject(block)
 
-    with nose.tools.assert_raises(TypeError):
+    flag = False
+    try:
         _ = subject.func_graph
+    except TypeError:
+        flag = True
+    assert flag
```

Comment:
Let's not reimplement `assertRaises`. Can you rewrite this function to use `unittest.TestCase` so you can call `self.assertRaises()` there?